citesphere/src/main/java/edu/asu/diging/citesphere/core/model/async/AsyncDeleteCitationsResponse.java

```diff
@@ -0,0 +1,38 @@
+package edu.asu.diging.citesphere.core.model.async;
```

Comment:
The model package should contain the main business logic classes (e.g. citation, person, etc). This class is only used to transfer information from zotero to the front end, isn't it? Therefore I would put it in the package or a subpackage with the corresponding service classes.